hw/bsp/nrf/linker/nrf52840_xxaa.ld

```diff
@@ -0,0 +1,20 @@
+/* Linker script to configure memory regions. */
+
+SEARCH_DIR(.)
+/*GROUP(-lgcc -lc -lnosys) not compatible with clang*/
```

Comment:
libgcc is not available in clang, since ld in nrfx/mdk contains this, we must keep an copy of lds just to remove this line